Classes/Command/ExtensionCommandController.php

```diff
@@ -161,6 +162,16 @@ public function dumpAutoloadCommand()
     }
 
     /**
+     * List all installed extensions
+     *
+     * @return void
+     */
+    public function listInstalledCommand()
+    {
+        $this->outputLine('%s', [implode(',', ExtensionManagementUtility::getLoadedExtensionListArray())]);
```

Comment:
We have a dependency to the package manager already in the class. please use it to get the active extensions instead of introducing the usage of a legacy API